src/content/api/loyalty-programs.mdx

```diff
@@ -0,0 +1,93 @@
+---
+title: Loyalty Programs
+description: Loyalty Program model and related endpoints
+nav:
+  path: API
+  order: 13
+---
+import Properties from '../../components/Properties.astro';
+import Property from '../../components/Property.astro';
+import Endpoint from '../../components/Endpoint.astro';
+import Badge from '../../components/Badge.astro';
+
+A loyalty program is a container for one or more user-facing components, to be surfaced in both Centrapay and third-party applications
```

Comment:
"A loyalty program is a container for one or more promotions, which is a mechanism to reward users for completing certain actions".

I know we don't have promotions documented yet but I think it's fine because we will be adding it very soon.

Reply:
Hey, you haven't resolved this - are you planning to change the line in the PR?

Reply:
@Ray2126  Thanks 😅 I realised I havent done anything to it apart from giving it a thumbs up. I've now updated it